vivisect/base.py

```diff
@@ -212,9 +216,23 @@ def _handleADDSEGMENT(self, einfo):
         self.segments.append(einfo)
 
     def _handleADDRELOC(self, einfo):
-        self.reloc_by_va[einfo[0]] = einfo[1]
+        rva, rtype = einfo
+        self.reloc_by_va[rva] = rtype
         self.relocations.append(einfo)
 
+        if rtype == RTYPE_BASERELOC:
+            fnm = self.getFileByVa(rva)
+            imgbase = self.getFileMeta(fnm, 'imagebase')
```

Comment:
Why not just `rbase = self.getFileMeta(fnm, 'imagebase')`?

Reply:
good catch.  i tried a couple different methods.  leftovers.

Reply:
stuck with imgbase, and did away with rbase.